Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,9 +115,30 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
+
+    // Parse command line options
+    String requestedURL = ""_s;
+
+    for (auto option : options) {
+        bool foundOption = false;
+        executeOption(option, &requestedURL);
+    }
+
     activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
 }
 
+void MainWindow::executeOption(const String& option, String* requestedURL)
```

Comment:
I'd like to move the for loop into here and rename `executeOptions` .
 `executeOptions`  sounds a bit weird to me. How about `parseOptions`?